crates/ruff_linter/src/rules/flake8_comprehensions/rules/unnecessary_dict_comprehension_for_iterable.rs

```diff
@@ -96,6 +96,12 @@ pub(crate) fn unnecessary_dict_comprehension_for_iterable(
         return;
     }
 
+    // Don't suggest `dict.fromkeys` if the target contains side-effecting expressions
+    // (attributes, subscripts, or slices) at the top level.
```

Comment:
```suggestion
    // (attributes, subscripts, or slices).
```

One more nit, I guess :) I think "at the top level" was only true before